lib/json_schema_form/umm_form.rb

```diff
@@ -329,7 +329,7 @@ def render_markup
 # :nodoc:
 class UmmFormAccordion < UmmForm
   def render_markup
-    content_tag(:fieldset, class: "eui-accordion is-closed #{parsed_json['htmlClass']}", id: accordion_id) do
+    content_tag(:fieldset, class: "eui-accordion is-closed", id: accordion_id) do
```

Comment:
I am thinking we may not want to move the `parsed_json['htmlClass']` from the top level tag for accordions, even though it might not be used at the moment. I am thinking we should use another key for the accordion header classes (something like `accordionHeaderClass` maybe) to be more specific about where they'll be applied.

Reply:
Okay, will do